contracts/test/TestInterchainGovernance.sol

```diff
@@ -0,0 +1,22 @@
+// SPDX-License-Identifier: MIT
+
+pragma solidity ^0.8.0;
+
+import { InterchainGovernance } from '../governance/InterchainGovernance.sol';
+
+contract TestInterchainGovernance is InterchainGovernance {
+    constructor(
+        address gatewayAddress,
+        string memory governanceChain_,
+        string memory governanceAddress_,
+        uint256 minimumTimeDelay
+    ) InterchainGovernance(gatewayAddress, governanceChain_, governanceAddress_, minimumTimeDelay) {}
+
+    function executeProposalAction(
```

Comment:
just override the `execute` method instead to skip gateway validation?

Reply:
execute is not overridable...